src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -107,6 +107,9 @@ The .NET Foundation licenses this file to you under the MIT license.
     <!-- Exclude System.IO.Compression.Native.dll -->
     <FrameworkAssemblies Include="$(IlcPath)\framework\*.dll" Exclude="$(IlcPath)\framework\*.Native.dll" />
   </ItemGroup>
+  <ItemGroup Condition="'$(IlcPgoOptimize)' == 'true'">
```

Comment:
Do we want this on by default?

Reply:
> I've seen this asserting the JIT

(There's something wrong with the PGO data I assume because the assert is in the form of `assert(haveBlockCounts != haveEdgeCounts);`)